### PR TITLE
mock global-agent/bootstrap in tests

### PR DESCRIPTION
**Pull request by jamieklassen**
## Hey, I just made a Pull Request!

Only changing tests. Related to #19915

#### :heavy_check_mark: Checklist

- [ ] ~A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))~  only tests
- [ ] ~Added or updated documentation~ N/A
- [ ] ~Tests for new functionality and regression tests for bug fixes~ N/A
- [ ] ~Screenshots attached (for UI changes)~ no UI
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
Oh interesting find.

Would it make sense to at least mention with a comment why that line is there and what it addresses? Maybe link to some article with info if one exists etc

File: packages/cli/src/commands/versions/bump.test.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ import { Command } from 'commander';
 import { resolve as resolvePath } from 'path';
 import { paths } from '../../lib/paths';
 import * as runObj from '../../lib/run';
+
+jest.mock('global-agent/bootstrap', () => {});
```
Comment (Rugvip) [inline packages/cli/src/commands/versions/bump.test.ts:24]: Still works if you move this down to after the imports right? It should be hoisted anyway.

Could you add a comment explaining what's going on as well? 🙏

**Review by Rugvip — CHANGES_REQUESTED** [review]
Awesome find! 😁 🎉

**Review by freben — APPROVED** [review]
<3

**Review by Rugvip — APPROVED** [review]
👌

**Pull request merged**